sys/unix/nledl.c

```diff
@@ -57,6 +60,16 @@ nle_ctx_t *
 nle_start(const char *dlpath, nle_obs *obs, FILE *ttyrec,
           nle_seeds_init_t *seed_init)
 {
+    static bool tc_started = FALSE;
+    if (!tc_started) {
```

Comment:
Why do we need this check?

Reply:
Otherwise we call `tgetent` several times again which we don't want to do.